Modules/posixmodule.c

```diff
@@ -12485,6 +12485,9 @@ check_ShellExecute()
 os.startfile
     filepath: path_t
     operation: Py_UNICODE = NULL
+    arguments: Py_UNICODE = NULL
+    cwd: object = NULL
```

Comment:
Wouldn't it be simpler to define this as `cwd: path_t(nullable=True) = None`?

Reply:
Yes, except it didn't work... but now it does, so I guess I'll switch.